joomla/joomla-cms
# [4.0][a11y] Improve accessiblity in mod_search

### Summary of Changes
Improve a11y by adding role="search" to the form tag.

### Testing Instructions
Add a search module to your website.
Inspect the souce code before and after patch.

see also https://issues.joomla.org/tracker/joomla-cms/23686
and good description in https://issues.joomla.org/tracker/joomla-cms/23692
